### PR TITLE
Work/tcomp 452 apply release plugin

### PR DESCRIPTION
**Pull request by gonchar-ivan**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Tests for the changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [x] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
Related other https://jira.talendforge.org/browse/TCOMP-452

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/25/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/25/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/29/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/29/)

File: pom.xml [hunk 1]
```diff
@@ -170,6 +186,16 @@
                     </execution>
                 </executions>
             </plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-release-plugin</artifactId>
+				<version>2.5.3</version>
+				<configuration>
+					<pushChanges>false</pushChanges>
+					<scmCommentPrefix>release:</scmCommentPrefix>
```
Comment (akhabali) [inline pom.xml]: just suggestion:
we may add here the artifact release version and a space after the prefix.
I don't know if we have rules for the release commit message but for the usual commits we use something like
**release(1.0.0):** update artifact release version

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/1/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/1/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/daikon_pull-request_master/2/) [:x:](https://ci-common.datapwn.com/job/daikon_pull-request_master/2/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/3/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/3/)

File: pom.xml (hunk 1 above)
Comment (akhabali) [inline pom.xml]: about the 
2) i was only talking about the prefix. the message was just to give an example of the final commit message style. Sorry if it was not clear

**Pull request merged**